index.html

```diff
@@ -12,81 +12,92 @@
 <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
 </head>
 <body>
-    <header>
-    <a href="#"> Ayodele </a>
-    <nav class="toggle">
-        <ul>
-        <li><i class="fas fa-bars"></i></li>
-        </ul>
-    </nav>
-    </header>
+    <div class="main">
+        <header>
+            <a href="#"> Ayodele </a>
+            <nav class="toggle">
+                <ul>
+                    <li><i class="fas fa-bars"></i></li>
+                </ul>
+            </nav>
+        </header>
     <section class="content">
         <h1>Hey There. <br> I’m Ayodele</h1>
         <h3>I’am a Software Developer</h3>
         <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-    <div class="item"><img src="img_social.png" alt="icons"></div>
-    <div id="background"></div>
+        <div class="item"><img src="img_social.png" alt="icons"></div>
```

Comment:
- [ ] [OPTIONAL] Please not that these icons are supposed to be links to your relevant pages, here however you have one `png`  that contains all icons, you can replace this line with icons that you have done for footer 🤗

Reply:
Thank you very much. I was able to make the icons in the rest of the work look like what i have in the footer of my work. Thank you. 😁